scheduler/views.py

```diff
@@ -225,7 +246,23 @@ def create(request: HttpRequest) -> HttpResponse:
                 start_time=form.cleaned_data['start_time'],
                 end_time=form.cleaned_data['end_time']
             )
+            db_conflicts()
             context = get_full_context_with_date(form.cleaned_data['start_time'])
+            current_conflicts = list(context['conflicts'])
+            print("was ", len(past_conflicts))
```

Comment:
To samo co w funkcji edit

Reply:
Tak było -> one znikają na Twoim branchu, bo w tym miałam zrobić coś co wypisuje te konflikty, albo robi z nimi cokolwiek...

Reply:
ale posprzątam :D